sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/AspNetHostedService/Directory.Build.props

```
@@ -0,0 +1,10 @@
+﻿<Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Import the common SDK build properties. -->
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\..\..\.., Directory.Build.props))\Directory.Build.props"
```

```suggestion
  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\..\.., Directory.Build.props))\Directory.Build.props"
```

If I'm reading correctly, I think this climbs 4 directories up, which puts us in the `/sdk` directory.  Since we want to be in `/sdk/eventhub`  I believe we need to trim one level.

Please double check me.

---

@jsquire  I can confirmed this is the correct change.